File: src/zenhan.py
# 全角撲滅
def zenhan(day):
    zen = "０１２３４５６７８９／（）、～　"
    han = "0123456789/(),~ "
    dic = dict(zip(zen, han))
    res = ''.join(dic.get(d, d) for d in day)
    return res

File: js/zenhan.js
function zenhan(day) {
    const zen = "０１２３４５６７８９／（）、～　";
    const han = "0123456789/(),~ ";
    const dic = Object.fromEntries([...zen].map((c, i) => [c, han[i]]));
    const res = [...day].map(c => dic[c] || c).join("");
    return res;
  }